api/init.lua

```diff
@@ -15979,6 +15999,33 @@ return {
           },
           notes = "The default clip distances are 0.1 and 100.0."
         },
+        {
+          name = "setDisplayFrequency",
+          tag = "headset",
+          summary = "Set the display refresh rate."
+          description = "Sets the display refresh rate, in Hz.",
+          key = "lovr.headset.setDisplayFrequency",
+          module = "lovr.headset",
+          variants = {
+            {
+              arguments = {
+                {
+                  name = "frequency",
+                  type = "number",
+                  description = "The refresh rate to set, in Hz",
+                }
+              },
+              returns = {
+                {
+                  name = "success",
+                  type = "boolean",
+                  description = "Whether the display refresh rate was successfully set."
+                }
+              }
+            }
+          },
+          notes = "Changing the display refresh-rate also changes the frequency of lovr.update() and lovr.draw() as they depend on the display frequency."
```

Comment:
I'm not sure whether it's worth adding; let me know if you think we should just remove this.